docs/backup/05-01-velero.md

```diff
@@ -35,6 +35,7 @@ This table lists the non-required configurable parameters, their descriptions, a
 
 | Parameter | Description | Default value |
 |-----------|-------------|---------------|
+| **schedules** | Sets up a scheduled backup | A scheduled backup runs at 07:00 on every day-of-week from Monday through Friday. |
```

Comment:
Maybe here we should also add the couple of values that can be set to change the scheduled backup, like `schedules.scheduled-backup.schedule` and `schedules.scheduled-backup.template.ttl`.